### ISSUE TITLE
Using Modelbuilder API issue

### ISSUE THREAD

**Issue by Dclapperton** [Gibe/Umbraco-Link-Picker#29]
Hey I'm using the Modelbuilder api and when I try to build the project the compiler is complaining that the namespace 'Gibe' can't be found despite the reference existing in the project.

Any idea what would cause this?

Thanks, David

**Comment by karltynan** [comment]
Not 100% sure as I would need @stevetemple to double check with me, but I think Nuget isn't including the Models and PropertyConverters - might be worth just checking you have all the appropriate cs files (https://github.com/Gibe/Umbraco-Link-Picker/tree/master/src/Gibe.LinkPicker.Umbraco) while I wait for Steve to input on his return?

**Comment by Dclapperton** [comment]
Hmm I think all the required cs files are in the dll. I'll try manually copy the files into my project for now.

Cheers!

**Comment by karltynan** [comment]
Ah yes, good point... Sorry, more front end and not been looking at this package recently. We can get back to you next week as this isn't my strong point.

**Comment by karltynan** [comment]
And especially not modelsbuilder...

**Comment by Dclapperton** [comment]
No you've been a great help I didn't know about PropertyConverters so that's going to save me a bunch of time thanks 😄
I've just removed the reference to the dll for now and moved the files into my project manually, seems to work!

Thanks

**Comment by karltynan** [comment]
Good start! We will have a look into it at some point to see if we can figure it out, but if you find a solution in the meantime, please feel free to contribute it back :)

**Comment by amrosama2020** [comment]
Had the same issue.
Fixed it without including the cs files since all required classes are included in DLL.

However to get strong type "_LinkPicker_" the following needs to be done:

1. Make sure you install via nuget to reference the DLL "Gibe.LinkPicker.Umbraco" in your project
2. Add the following line to the views folder web.config "<add namespace="Gibe.LinkPicker.Umbraco" />" (would be nice if this change gets added to nuget package)
3. Add a using to your cshtml file "@using Gibe.LinkPicker.Umbraco.Models"

I tried this on both regular views and macros
